pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -314,6 +324,23 @@ func (self *FakeMetadata) GetMetaData(key string) ([]byte, error) {
 		return []byte(self.aws.instanceId), nil
 	} else if key == "local-hostname" {
 		return []byte(self.aws.privateDnsName), nil
+	} else if key == "network/interfaces/macs/" {
+		macs := fmt.Sprintf(
+			"%s/\n%s/\n",
+			self.aws.networkInterfacesMac0,
+			self.aws.networkInterfacesMac1,
+		)
+		return []byte(macs), nil
+	} else if key == fmt.Sprintf(
+		"network/interfaces/macs/%s/vpc-id",
+		self.aws.networkInterfacesMac0,
+	) {
+		return []byte(self.aws.networkInterfacesMac0VpcId), nil
+	} else if key == fmt.Sprintf(
+		"network/interfaces/macs/%s/vpc-id",
+		self.aws.networkInterfacesMac1,
```

Comment:
The above approach would also let you replace networkInterfacesMac0 and networkInterfacesMac1 with `networkInterfacesMac []string`.  Although as this is test code I don't think it matters as much - your call!